README.md

```diff
@@ -79,21 +88,123 @@ POINTS
 1 0 0
 
 ```
+Parameters could be passed as
+* keyword arguments (as above),
+* `Pair{Symbol, ...}`s e.g. `Polytope.Polytope(:POINTS=>[ ... ])`
+* dictionaries e.g. `Polytope.Polytope(Dict( "POINTS" => [ ... ] )`)
 
-Properties of such objects can be accessed by the `.` syntax
+The dictionary may hold many different attributes. All the names *must* be compatible with `perl/polymake`.
+
+Properties of such objects can be accessed by the `.` syntax:
 ```
 julia> p.INTERIOR_LATTICE_POINTS
 pm::Matrix<pm::Integer>
 1 0 0
 ```
 
+### Example script
+
+The following script is modelled on the one from the [Using Perl within polymake](https://polymake.org/doku.php/user_guide/tutorials/perl_intro) tutorial:
+
+```julia
+using Polymake
+
+str = read("points.demo", String)
+matrix_str = "["*replace(str, "/"=>"//")*"]"
+matrix = eval(Base.Meta.parse(matrix_str))
```

Comment:
Do we really need an `eval` here? Or maybe we should provide a helper function to hide this?

Reply:
I don't know actually what is the best way to do so, but this one was the shortest (e.g. there is no `Base.parse(::Type{Rational{T}, ...)`),
```
matrix = Meta.parse(matrix_str) |> eval
```
??? (more obscure, I'd say)

Reply:
Yeah I thought one can use `DelimitedFiles` but this is not the case :(

Reply:
Maybe just a comment that this is a hack?^^